hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/ddl/TestAlterTableDropPartition.scala

```diff
@@ -133,11 +141,13 @@ class TestAlterTableDropPartition extends HoodieSparkSqlTestBase {
 
         // drop 2021-10-01 partition
         spark.sql(s"alter table $tableName drop partition (dt='2021/10/01')")
+        ensureDataCanBeReadInIncrementalQuery(tablePath)
```

Comment:
Instead of validation at the Spark read level, could you add the validation on the schema in the commit metadata only?

Reply:
I will do that